### ISSUE TITLE
Support CSS #rrggbbaa format 

### ISSUE THREAD

**Issue by okonet** [styled-components/polished#306]
- `polished` version: 1.9.2
- `JSS-in_CSS` library and version: styled-components 3.2.5
- Any relevant JS library and version: React 16.3.0

Mixin/Helper/Shorthand you were using and how you were using it:

```javascript
import { darken } from 'polished';
darken(0.5, '#FF0000CC');
```

What you are seeing:

```
polished.es.js:1461 Uncaught Error: Couldn't parse the color string. Please provide the color as a string in hex, rgb, rgba, hsl or hsla notation.
```

What you expected to see:

The color should be properly parsed since it's a valid color.

Reproduction:

https://codesandbox.io/s/50mlvjz2l4

**Comment by bhough** [comment]
Thanks @okonet I will look at possibly getting this into the 2.0 release.

**Comment by bhough** [comment]
Addressed by #318 and merged into `version-2`
